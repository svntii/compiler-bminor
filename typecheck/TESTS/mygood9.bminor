// print something


x:function integer(i:function integer) = {
    return x;
}