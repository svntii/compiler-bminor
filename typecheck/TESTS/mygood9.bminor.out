	Scan Successful
	Parse Successful
	Reformatted Code
x: function integer(i: function integer)  = 
{
    return x;
    
}


*****RESOLVING*******

i resolves to (param)
x is bound to (global) x
	Code Resolved

*****TYPECHECKING*******

	Code Typecheck
